src/main/java/org/apache/cassandra/distributed/api/NodeToolResult.java

```diff
@@ -134,6 +144,30 @@ public Asserts notificationContains(ProgressEventType type, String msg)
             return this; // unreachable
         }
 
+        public Asserts capturedConsoleOutContains(String substring)
```

Comment:
maybe `stdoutContains` and `stderrContains`?  `capturedConsole` I don't feel is needed in the name